spec.html

```diff
@@ -38330,7 +38330,7 @@ <h1>Runtime Semantics: PerformPromiseAll ( _iteratorRecord_, _constructor_, _res
               1. Append *undefined* to _values_.
               1. Let _nextPromise_ be ? Call(_promiseResolve_, _constructor_, &laquo; _nextValue_ &raquo;).
               1. Let _steps_ be the algorithm steps defined in <emu-xref href="#sec-promise.all-resolve-element-functions" title></emu-xref>.
-              1. Let _resolveElement_ be CreateBuiltinFunction(_steps_, &laquo; [[AlreadyCalled]], [[Index]], [[Values]], [[Capability]], [[RemainingElements]] &raquo;).
+              1. Let _resolveElement_ be ! CreateBuiltinFunction(_steps_, &laquo; [[AlreadyCalled]], [[Index]], [[Values]], [[Capability]], [[RemainingElements]] &raquo;).
```

Comment:
This can throw, because it’s calling user code in step 11b of https://tc39.es/ecma262/#sec-promise.all-resolve-element-functions

Reply:
I don't believe this is correct. CreateBuiltinFunction just creates a function and assigns those steps to it. It doesn't execute the function it creates.

Reply:
ah, that’s true - in that case, it seems like every call to CreateBuiltinFunction should have `!`?

Reply:
In this particular case it can fail on `Promise.all` Resolve Element Function

Reply:
It can fail because of previous step (Promise.all Resolve Element Functions in this case)

Reply:
Those steps won’t be invoked here tho, per https://github.com/tc39/ecma262/pull/1607#discussion_r299503987 - just creating the function doesn’t seem to ever fail.

Reply:
How to indicate that 
```
Let _steps_ be the algorithm steps defined in <emu-xref href="#sec-promise.all-resolve-element-functions" title></emu-xref>.
```
may fail?

Reply:
That can never fail. That's just letting _steps_ have a certain value.

Reply:
I've replaced all `?` usages by `!` for `CreateBuiltinFunction`